src/pages/NotificationUserConfigPage/hooks/useNotificationConfigHook.ts

```diff
@@ -0,0 +1,256 @@
+import { useTranslation } from 'react-i18next'
+import type { NotificationSubSectionSchema } from '../types'
+import { type NotificationConfigSchema, type NotificationConfigType } from '../types'
+import { match } from 'ts-pattern'
+import React from 'react'
+import { ConsumerIcon, ProviderIcon, MyTenantIcon } from '@/icons'
+import CodeIcon from '@mui/icons-material/Code'
+
+export function useNotificationConfigHook(type: NotificationConfigType) {
```

Comment:
The `Hook` suffix communicates nothing, can you find a better one? What about `useGetNotificationConfigSchema`?